homeassistant/components/mpd/media_player.py

```diff
@@ -251,6 +252,30 @@ def media_album_name(self):
         """Return the album of current playing media (Music track only)."""
         return self._currentsong.get("album")
 
+    @property
+    def media_image_hash(self):
+        """Hash value for media image."""
+        file = self._currentsong.get("file", None)
+        if file:
+            return hashlib.sha256(file.encode("utf-8")).hexdigest()[:16]
+
+        return None
+
+    async def async_get_media_image(self):
+        """Fetch media image of current playing track."""
+        # not all MPD implementations support the `albumart` command
+        if "albumart" not in self._client.commands():
```

Comment:
Does this do I/O?

Reply:
Yes, communicates with mpd for available commands. 

I agree this is somewhat noisy and we could probably get away with querying commands once in the constructor.

Reply:
Fetching commands directly after connect, so we will pick up new commands, when mpd restarts etc..